flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/xa/XaFacade.java

```diff
@@ -52,8 +51,8 @@
 
     /** @return a non-serializable instance. */
```

Comment:
This is only for test?

Mark as visible for test?

Reply:
and other similar places.

Reply:
This particular method is used in `JdbcSink#exactlyOnceSink`.
The comment is misleading however; I'll remove it.